<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_SplineFromPolylineOptions</class>
 <widget class="QWidget" name="LC_SplineFromPolylineOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="10">
    <widget class="QCheckBox" name="cbLayer">
     <property name="toolTip">
      <string>If checked, created entities will be placed on current layer, otherwise they will be placed into original layers.</string>
     </property>
     <property name="text">
      <string>User Current Layer</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QSpinBox" name="sbMidPoints">
     <property name="toolTip">
      <string>Amount of middle points of polyline segment that will be added to spline. </string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="sbDegree">
     <property name="toolTip">
      <string>Defines degree of the spline</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QCheckBox" name="cbKeepOriginals">
     <property name="toolTip">
      <string>If unchecked, original polyline will be removed, otherwise it will survive.</string>
     </property>
     <property name="text">
      <string>Keep Originals</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Mid points:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QCheckBox" name="cbCurrentAttr">
     <property name="toolTip">
      <string>If checked, current attributes will be used for created entities, otherwise - original ones. </string>
     </property>
     <property name="text">
      <string>Use Current Attributes</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Degree:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="cbFitPoints">
     <property name="toolTip">
      <string>If checked, polyline vertexes are considered as spline points and spline by points will be created. Otherwise, they are considered as control points and orinary spline will be created.</string>
     </property>
     <property name="text">
      <string>Vertexes are spline points</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
